packages/next/src/build/index.ts

```diff
@@ -395,7 +395,9 @@ export default async function build(
           // Intentionally not piping to stderr in case people fail in CI when
           // stderr is detected.
           console.log(
-            `${Log.prefixes.warn} No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache`
+            `${Log.createPrefix(
+              'warn'
```

Comment:
Would recommend changing these to an enum so that renaming and finding all usage of it is easier later on:

```
export const enum LOG_LEVELS = {
  WARN = 'warn',
  // etc
}
```

Reply:
Seems you already defined something like that here: https://github.com/vercel/next.js/pull/54713/files#diff-5a12bda99653661a0ca55716e03cbf6895bd1a6f9f7eb7b6c9303e2ee16a7819R13